subsys/bluetooth/mesh/net.c

```diff
@@ -283,6 +283,11 @@ bool bt_mesh_net_iv_update(uint32_t iv_index, bool iv_update)
 		return false;
 	}
 
+	/* Discard [iv, false] --> [iv, true] */
+	if (iv_index == bt_mesh.iv_index && iv_update) {
+		return false;
```

Comment:
Could you add any `LOG_DBG` or `LOG_ERR` message before `return false`? 
IV update procedure is quite difficult to debug\investigate. It will help in maintenance.